rules_gapic/bazel/src/main/java/com/google/api/codegen/bazel/BUILD.bazel.gapic_api.mustache

```diff
@@ -106,10 +106,9 @@ go_proto_library(
 
 go_gapic_library(
     name = "{{name}}_go_gapic",
-    src = ":{{name}}_proto_with_info",
-    gapic_yaml = "{{gapic_yaml}}",
-    importpath = "{{go_gapic_importpath}}",
-    package = "{{package}}",
+    srcs = [":{{name}}_proto_with_info"],
+    grpc_service_config = "{{grpc_service_config}}",
+    importpath = "{{go_gapic_importpath_pkg}}",
```

Comment:
Should I use `{{name}}` attribute here to add the package name suffix instead of parsing the proto package using the newly added `assembleGoGapicImportPath` method?

`importpath = "{{go_gapic_importpath}};{{name}}",`

Reply:
`{{name}}` is derived in the same way, so I'm going to use that instead of redoing the same parsing with a helper function.